R/install-dev.sh

```diff
@@ -35,12 +35,19 @@ LIB_DIR="$FWDIR/lib"
 mkdir -p $LIB_DIR
 
 pushd $FWDIR > /dev/null
+if [ ! -z "$1" ]
+  then
+    R_HOME="$1/bin"
+   else
+    R_HOME="$(dirname $(which R))"
```

Comment:
Can we just use a `R_HOME` if it is set instead of using a command line argument ?